portfolio/src/main/webapp/index.html

```diff
@@ -76,14 +76,58 @@ <h1>About</h1>
                 </div>
             </div>
         </section>
+        <!--========= Projects Section ========-->
+       <section class="projects" id="projects">
+           <div class="container">
+               <div class="section-title">
+                   <h1>Projects</h1>
+               </div>
+               <div class="row d-flex justify-content-center">
+                   <div class="col-md-4">
+                       <div class="card" style="width: 18rem;">
+                       <img class="card-img-top" src="img/project-1.jpeg" alt="Card image cap">
```

Comment:
I would name the file for these images a more descriptively (e.g. "img/budget-project.jpeg", "img/simon-says-project") so that way its easier to remember/tell what the image is without having to go back and look at it.